apps/dashboard/src/components/primitives/field-editor/variable-popover/components/transformer-item.tsx

```diff
@@ -0,0 +1,27 @@
+import TruncatedText from '@/components/truncated-text';
+import { Transformer } from '../types';
+
+type TransformerItemProps = {
+  transformer: Transformer;
+};
+
+export function TransformerItem({ transformer }: TransformerItemProps) {
```

Comment:
As per Liquid.js jargon and [Novu Lexicon](https://www.notion.so/novuhq/Product-Terminology-Lexicon-d6ef923953744bc1b967ba8f297dee7f?pvs=4#15bd03ad838f8070a697c74fa7f8efa7) these are Liquid Filters, not transformers.

I strongly advice aligning the terminology across the board to easy the future reader.